debian/precise/foreman/foreman-debug.dirs

```diff
@@ -0,0 +1 @@
+usr/share/foreman/script/foreman-debug.d/
```

Comment:
Katello and plugins use that to drop "extensions" to the foreman-debug to collect more information. Let's move the review to the new PR, had to change my branch name.